fedimint-server/src/consensus/server.rs

```diff
@@ -856,7 +863,7 @@ async fn submit_module_consensus_items(
                     }
                 }
 
-                sleep(Duration::from_secs(1)).await;
+                interval.tick().await;
```

Comment:
[nit] Since the first tick completes immediately, we can move to the top of the loop (same for other usages of `tick` in this PR)